test/linalg/conjarray.jl

```diff
@@ -0,0 +1,23 @@
+# This file is a part of Julia. License is MIT: http://julialang.org/license
+
+@testset "RowVector" begin
```

Comment:
don't think this outer testset is necessary, the test system inserts a test set around each file anyway